doc-site/sidebars.js

```diff
@@ -0,0 +1,58 @@
+const config = require('./docusaurus.config');
+
+module.exports = {
+  sidebar: [
+    {
+      "About": [
+        {
+          type: 'autogenerated',
+          dirName: 'about',
+        },
+      ],
+      "Components & References": [
+        {
+          type: 'autogenerated',
+          dirName: 'components',
+        },
+      ],
+      "Guides": [
+        {
+          type: 'autogenerated',
+          dirName: 'guides',
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Modules',
+      collapsed: false,
+      items: [
+        {
+          type: 'link',
+          label: 'Zero Core',
+          href: `${config.url}/docs/zero`,
+        },
+        {
+          type: 'link',
+          label: 'AWS EKS stack',
+          href: `${config.url}/docs/modules/aws-eks-stack`,
+        },
+        {
+          type: 'link',
+          label: 'Frontend',
+          href: `${config.url}/docs/modules/frontend-react`,
+        },
+        {
+          type: 'link',
+          label: 'Backend - Go',
+          href: `${config.url}/docs/modules/backend-go`,
+        },
+        {
+          type: 'link',
+          label: 'Backend - Node.js',
+          href: `${config.url}/docs/modules/backend-nodejs`,
+        }
```

Comment:
I'm thinking of a way to consolidate all these repeated and should stay the same UI components, perhaps make a common npm package and use that.